static/ankersrv.css

```diff
@@ -0,0 +1,41 @@
+.nav-link,
+.btn-link {
+  color: #88f387;
+  text-decoration: none;
+}
+
+.nav-link.active,
+.btn-primary {
+  color: #000000 !important;
```

Comment:
Nit picking. `!important` should be avoided if at all possible and handled via specificity. You can leave as is and I can clean up later though when I get a chance. It does work